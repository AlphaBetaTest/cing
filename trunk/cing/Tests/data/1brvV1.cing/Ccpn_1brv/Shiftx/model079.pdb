REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       3.953   2.069   5.559  1.00  0.00              
ATOM      2  CA  VAL A 171       3.724   2.125   4.095  1.00  0.00              
ATOM      3  HA  VAL A 171       3.392   3.147   3.926  1.00  0.00              
ATOM      4  CB  VAL A 171       5.099   1.937   3.369  1.00  0.00              
ATOM      5  HB  VAL A 171       4.883   2.199   2.330  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.227   2.916   3.770  1.00  0.00              
ATOM      7 HG11 VAL A 171       5.888   3.967   3.717  1.00  0.00              
ATOM      8 HG12 VAL A 171       6.591   2.738   4.799  1.00  0.00              
ATOM      9 HG13 VAL A 171       7.102   2.829   3.097  1.00  0.00              
ATOM     10  CG2 VAL A 171       5.659   0.498   3.354  1.00  0.00              
ATOM     11 HG21 VAL A 171       4.917  -0.223   2.961  1.00  0.00              
ATOM     12 HG22 VAL A 171       6.550   0.414   2.704  1.00  0.00              
ATOM     13 HG23 VAL A 171       5.943   0.153   4.365  1.00  0.00              
ATOM     14  C   VAL A 171       2.566   1.328   3.384  1.00  0.00              
ATOM     15  O   VAL A 171       2.201   1.806   2.310  1.00  0.00              
ATOM     16  N   PRO A 172       1.963   0.190   3.841  1.00  0.00              
ATOM     17  CD  PRO A 172       2.266  -0.384   5.150  1.00  0.00              
ATOM     18  CA  PRO A 172       0.918  -0.568   3.090  1.00  0.00              
ATOM     19  HA  PRO A 172       1.441  -1.127   2.288  1.00  0.00              
ATOM     20  CB  PRO A 172       0.431  -1.575   4.154  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.426  -1.168   4.729  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.088  -2.527   3.707  1.00  0.00              
ATOM     23  CG  PRO A 172       1.620  -1.766   5.096  1.00  0.00              
ATOM     24  HG2 PRO A 172       1.324  -2.131   6.097  1.00  0.00              
ATOM     25  HG3 PRO A 172       2.333  -2.502   4.675  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.826   0.288   5.918  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.359  -0.430   5.318  1.00  0.00              
ATOM     28  C   PRO A 172      -0.264   0.240   2.467  1.00  0.00              
ATOM     29  O   PRO A 172      -0.869   1.084   3.133  1.00  0.00              
ATOM     30  N   CYS A 173      -0.617  -0.048   1.205  1.00  0.00              
ATOM     31  H   CYS A 173      -0.033  -0.725   0.705  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.673   0.685   0.457  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.413   1.746   0.426  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.544   0.244  -1.003  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.939  -0.657  -1.145  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.542   0.068  -1.392  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.661   1.464  -1.978  1.00  0.00              
ATOM     38  C   CYS A 173      -3.108   0.659   1.088  1.00  0.00              
ATOM     39  O   CYS A 173      -3.809   1.675   1.046  1.00  0.00              
ATOM     40  N   SER A 174      -3.509  -0.461   1.720  1.00  0.00              
ATOM     41  H   SER A 174      -2.789  -1.191   1.761  1.00  0.00              
ATOM     42  CA  SER A 174      -4.715  -0.502   2.601  1.00  0.00              
ATOM     43  HA  SER A 174      -5.577  -0.207   1.999  1.00  0.00              
ATOM     44  CB  SER A 174      -4.927  -1.958   3.073  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.849  -2.006   3.688  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.110  -2.615   2.204  1.00  0.00              
ATOM     47  OG  SER A 174      -3.818  -2.452   3.836  1.00  0.00              
ATOM     48  HG  SER A 174      -4.111  -3.249   4.291  1.00  0.00              
ATOM     49  C   SER A 174      -4.746   0.479   3.827  1.00  0.00              
ATOM     50  O   SER A 174      -5.819   0.792   4.344  1.00  0.00              
ATOM     51  N   THR A 175      -3.574   0.952   4.265  1.00  0.00              
ATOM     52  H   THR A 175      -2.763   0.555   3.774  1.00  0.00              
ATOM     53  CA  THR A 175      -3.405   1.927   5.381  1.00  0.00              
ATOM     54  HA  THR A 175      -4.363   1.991   5.933  1.00  0.00              
ATOM     55  CB  THR A 175      -2.354   1.358   6.386  1.00  0.00              
ATOM     56  HB  THR A 175      -2.358   2.013   7.281  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.030   1.365   5.851  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.097   1.284   4.889  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.655  -0.081   6.856  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.689  -0.169   7.239  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.581  -0.801   6.016  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.957  -0.417   7.636  1.00  0.00              
ATOM     63  C   THR A 175      -3.095   3.401   4.919  1.00  0.00              
ATOM     64  O   THR A 175      -3.307   4.335   5.695  1.00  0.00              
ATOM     65  N   CYS A 176      -2.603   3.630   3.679  1.00  0.00              
ATOM     66  H   CYS A 176      -2.664   2.831   3.042  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.424   4.977   3.083  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.904   5.636   3.806  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.482   4.702   1.889  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.869   4.008   1.150  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.300   5.579   1.248  1.00  0.00              
ATOM     72  SG  CYS A 176       0.103   4.109   2.510  1.00  0.00              
ATOM     73  C   CYS A 176      -3.784   5.676   2.733  1.00  0.00              
ATOM     74  O   CYS A 176      -3.941   6.879   2.947  1.00  0.00              
ATOM     75  N   GLU A 177      -4.743   4.912   2.173  1.00  0.00              
ATOM     76  H   GLU A 177      -4.406   3.998   1.863  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.165   5.309   1.971  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.550   4.534   1.295  1.00  0.00              
ATOM     79  CB  GLU A 177      -6.937   5.178   3.322  1.00  0.00              
ATOM     80  HB2 GLU A 177      -6.488   5.873   4.060  1.00  0.00              
ATOM     81  HB3 GLU A 177      -7.973   5.548   3.188  1.00  0.00              
ATOM     82  CG  GLU A 177      -7.013   3.749   3.921  1.00  0.00              
ATOM     83  HG2 GLU A 177      -7.509   3.057   3.218  1.00  0.00              
ATOM     84  HG3 GLU A 177      -5.995   3.343   4.060  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.741   3.661   5.257  1.00  0.00              
ATOM     86  OE1 GLU A 177      -8.958   3.567   5.369  1.00  0.00              
ATOM     87  OE2 GLU A 177      -6.888   3.693   6.316  1.00  0.00              
ATOM     88  C   GLU A 177      -6.459   6.631   1.202  1.00  0.00              
ATOM     89  O   GLU A 177      -7.271   7.470   1.599  1.00  0.00              
ATOM     90  N   GLY A 178      -5.810   6.755   0.042  1.00  0.00              
ATOM     91  H   GLY A 178      -4.925   6.240   0.070  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.911   7.972  -0.788  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.954   7.631  -1.833  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.864   8.481  -0.607  1.00  0.00              
ATOM     95  C   GLY A 178      -4.773   9.011  -0.682  1.00  0.00              
ATOM     96  O   GLY A 178      -4.816  10.017  -1.392  1.00  0.00              
ATOM     97  N   ASN A 179      -3.745   8.780   0.157  1.00  0.00              
ATOM     98  H   ASN A 179      -3.676   7.805   0.466  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.565   9.656   0.238  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.821  10.685  -0.080  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.924   9.551   1.636  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.669   9.373   2.437  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.273   8.651   1.590  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.028  10.730   1.977  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.119  10.813   1.557  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.529  11.687   2.707  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.490  11.553   3.034  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.878  12.449   2.904  1.00  0.00              
ATOM    109  C   ASN A 179      -1.575   9.129  -0.785  1.00  0.00              
ATOM    110  O   ASN A 179      -0.926   8.093  -0.605  1.00  0.00              
ATOM    111  N   LEU A 180      -1.442   9.870  -1.862  1.00  0.00              
ATOM    112  H   LEU A 180      -2.078  10.667  -1.947  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.634   9.399  -2.980  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.048   8.385  -3.069  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.026  10.343  -4.127  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.195  11.369  -3.722  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.171  10.402  -4.770  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.198   9.919  -5.018  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.004   9.734  -4.310  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.615  11.056  -5.963  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.868  11.982  -5.413  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.815  11.313  -6.684  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.511  10.789  -6.554  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.924   8.645  -5.839  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.683   7.775  -5.202  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.801   8.353  -6.446  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.076   8.776  -6.538  1.00  0.00              
ATOM    128  C   LEU A 180       0.897   9.182  -2.763  1.00  0.00              
ATOM    129  O   LEU A 180       1.509   8.351  -3.430  1.00  0.00              
ATOM    130  N   ALA A 181       1.466   9.902  -1.799  1.00  0.00              
ATOM    131  H   ALA A 181       0.741  10.443  -1.328  1.00  0.00              
ATOM    132  CA  ALA A 181       2.824   9.647  -1.273  1.00  0.00              
ATOM    133  HA  ALA A 181       3.507   9.581  -2.134  1.00  0.00              
ATOM    134  CB  ALA A 181       3.215  10.875  -0.445  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.184  11.800  -1.048  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.545  11.009   0.427  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.243  10.764  -0.055  1.00  0.00              
ATOM    138  C   ALA A 181       2.919   8.300  -0.469  1.00  0.00              
ATOM    139  O   ALA A 181       3.762   7.468  -0.792  1.00  0.00              
ATOM    140  N   CYS A 182       2.031   8.060   0.522  1.00  0.00              
ATOM    141  H   CYS A 182       1.214   8.681   0.474  1.00  0.00              
ATOM    142  CA  CYS A 182       1.914   6.753   1.240  1.00  0.00              
ATOM    143  HA  CYS A 182       2.793   6.500   1.795  1.00  0.00              
ATOM    144  CB  CYS A 182       0.890   7.000   2.342  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.059   7.955   2.845  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.107   7.084   1.913  1.00  0.00              
ATOM    147  SG  CYS A 182       0.860   5.643   3.521  1.00  0.00              
ATOM    148  C   CYS A 182       1.676   5.523   0.315  1.00  0.00              
ATOM    149  O   CYS A 182       2.394   4.522   0.355  1.00  0.00              
ATOM    150  N   LEU A 183       0.680   5.669  -0.550  1.00  0.00              
ATOM    151  H   LEU A 183       0.164   6.554  -0.450  1.00  0.00              
ATOM    152  CA  LEU A 183       0.373   4.743  -1.652  1.00  0.00              
ATOM    153  HA  LEU A 183       0.067   3.772  -1.230  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.826   5.456  -2.310  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.530   6.481  -2.580  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.986   5.015  -3.250  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.212   5.441  -1.659  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.056   5.936  -0.685  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.190   6.233  -2.543  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.186   5.873  -3.591  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.229   6.148  -2.190  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.945   7.308  -2.577  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.778   4.019  -1.506  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.116   3.377  -0.909  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.761   4.009  -1.003  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.898   3.512  -2.483  1.00  0.00              
ATOM    167  C   LEU A 183       1.524   4.479  -2.695  1.00  0.00              
ATOM    168  O   LEU A 183       1.599   3.391  -3.264  1.00  0.00              
ATOM    169  N   SER A 184       2.381   5.479  -2.962  1.00  0.00              
ATOM    170  H   SER A 184       2.179   6.289  -2.358  1.00  0.00              
ATOM    171  CA  SER A 184       3.616   5.305  -3.797  1.00  0.00              
ATOM    172  HA  SER A 184       3.343   4.752  -4.718  1.00  0.00              
ATOM    173  CB  SER A 184       4.163   6.681  -4.246  1.00  0.00              
ATOM    174  HB2 SER A 184       4.418   7.310  -3.371  1.00  0.00              
ATOM    175  HB3 SER A 184       5.112   6.545  -4.800  1.00  0.00              
ATOM    176  OG  SER A 184       3.242   7.366  -5.095  1.00  0.00              
ATOM    177  HG  SER A 184       2.502   7.642  -4.528  1.00  0.00              
ATOM    178  C   SER A 184       4.745   4.454  -3.109  1.00  0.00              
ATOM    179  O   SER A 184       5.432   3.687  -3.787  1.00  0.00              
ATOM    180  N   LEU A 185       4.953   4.596  -1.785  1.00  0.00              
ATOM    181  H   LEU A 185       4.283   5.232  -1.328  1.00  0.00              
ATOM    182  CA  LEU A 185       5.883   3.716  -0.997  1.00  0.00              
ATOM    183  HA  LEU A 185       6.819   3.557  -1.536  1.00  0.00              
ATOM    184  CB  LEU A 185       6.163   4.368   0.394  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.259   4.226   1.018  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.953   3.782   0.903  1.00  0.00              
ATOM    187  CG  LEU A 185       6.535   5.865   0.464  1.00  0.00              
ATOM    188  HG  LEU A 185       5.691   6.401  -0.011  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.633   6.343   1.920  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.686   6.179   2.466  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.432   5.820   2.478  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.846   7.427   1.980  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.843   6.189  -0.276  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.781   5.917  -1.346  1.00  0.00              
ATOM    195 HD22 LEU A 185       8.078   7.269  -0.239  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.708   5.648   0.150  1.00  0.00              
ATOM    197  C   LEU A 185       5.437   2.241  -0.763  1.00  0.00              
ATOM    198  O   LEU A 185       6.254   1.382  -0.412  1.00  0.00              
ATOM    199  N   CYS A 186       4.144   1.974  -0.927  1.00  0.00              
ATOM    200  H   CYS A 186       3.585   2.818  -0.805  1.00  0.00              
ATOM    201  CA  CYS A 186       3.547   0.704  -0.588  1.00  0.00              
ATOM    202  HA  CYS A 186       3.960   0.522   0.421  1.00  0.00              
ATOM    203  CB  CYS A 186       2.035   1.038  -0.546  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.611   0.331   0.163  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.731   2.014  -0.145  1.00  0.00              
ATOM    206  SG  CYS A 186       1.225   0.829  -2.144  1.00  0.00              
ATOM    207  C   CYS A 186       3.755  -0.573  -1.456  1.00  0.00              
ATOM    208  O   CYS A 186       4.160  -0.592  -2.620  1.00  0.00              
ATOM    209  N   HIS A 187       3.308  -1.651  -0.815  1.00  0.00              
ATOM    210  H   HIS A 187       3.497  -1.541   0.187  1.00  0.00              
ATOM    211  CA  HIS A 187       3.189  -2.996  -1.418  1.00  0.00              
ATOM    212  HA  HIS A 187       3.863  -3.094  -2.286  1.00  0.00              
ATOM    213  CB  HIS A 187       3.611  -4.090  -0.404  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.786  -4.280   0.274  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.721  -5.031  -0.953  1.00  0.00              
ATOM    216  CG  HIS A 187       4.854  -3.861   0.445  1.00  0.00              
ATOM    217  ND1 HIS A 187       4.871  -3.027   1.552  1.00  0.00              
ATOM    218  CD2 HIS A 187       6.140  -4.145   0.008  1.00  0.00              
ATOM    219  CE1 HIS A 187       6.221  -2.839   1.650  1.00  0.00              
ATOM    220  NE2 HIS A 187       7.072  -3.498   0.800  1.00  0.00              
ATOM    221  HD2 HIS A 187       6.311  -4.658  -0.922  1.00  0.00              
ATOM    222  HE1 HIS A 187       6.568  -2.030   2.271  1.00  0.00              
ATOM    223  C   HIS A 187       1.691  -3.174  -1.856  1.00  0.00              
ATOM    224  O   HIS A 187       0.767  -3.142  -1.032  1.00  0.00              
ATOM    225  N   ILE A 188       1.449  -3.375  -3.156  1.00  0.00              
ATOM    226  H   ILE A 188       2.291  -3.356  -3.736  1.00  0.00              
ATOM    227  CA  ILE A 188       0.066  -3.486  -3.720  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.662  -3.006  -3.038  1.00  0.00              
ATOM    229  CB  ILE A 188      -0.101  -2.675  -5.057  1.00  0.00              
ATOM    230  HB  ILE A 188       0.588  -3.131  -5.787  1.00  0.00              
ATOM    231  CG2 ILE A 188      -1.534  -2.812  -5.642  1.00  0.00              
ATOM    232 HG21 ILE A 188      -1.814  -3.865  -5.824  1.00  0.00              
ATOM    233 HG22 ILE A 188      -2.304  -2.380  -4.974  1.00  0.00              
ATOM    234 HG23 ILE A 188      -1.630  -2.307  -6.622  1.00  0.00              
ATOM    235  CG1 ILE A 188       0.203  -1.143  -4.974  1.00  0.00              
ATOM    236 HG12 ILE A 188      -0.350  -0.586  -5.757  1.00  0.00              
ATOM    237 HG13 ILE A 188      -0.184  -0.731  -4.023  1.00  0.00              
ATOM    238  CD1 ILE A 188       1.680  -0.749  -5.152  1.00  0.00              
ATOM    239 HD11 ILE A 188       2.093  -1.102  -6.115  1.00  0.00              
ATOM    240 HD12 ILE A 188       1.807   0.351  -5.133  1.00  0.00              
ATOM    241 HD13 ILE A 188       2.331  -1.140  -4.351  1.00  0.00              
ATOM    242  C   ILE A 188      -0.275  -5.014  -3.841  1.00  0.00              
ATOM    243  O   ILE A 188       0.139  -5.682  -4.792  1.00  0.00              
ATOM    244  N   GLU A 189      -1.029  -5.548  -2.859  1.00  0.00              
ATOM    245  H   GLU A 189      -1.162  -4.900  -2.076  1.00  0.00              
ATOM    246  CA  GLU A 189      -1.417  -6.992  -2.763  1.00  0.00              
ATOM    247  HA  GLU A 189      -1.764  -7.112  -1.720  1.00  0.00              
ATOM    248  CB  GLU A 189      -2.686  -7.316  -3.615  1.00  0.00              
ATOM    249  HB2 GLU A 189      -3.009  -8.345  -3.361  1.00  0.00              
ATOM    250  HB3 GLU A 189      -3.514  -6.675  -3.255  1.00  0.00              
ATOM    251  CG  GLU A 189      -2.617  -7.194  -5.163  1.00  0.00              
ATOM    252  HG2 GLU A 189      -2.360  -6.162  -5.458  1.00  0.00              
ATOM    253  HG3 GLU A 189      -1.803  -7.817  -5.571  1.00  0.00              
ATOM    254  CD  GLU A 189      -3.902  -7.596  -5.876  1.00  0.00              
ATOM    255  OE1 GLU A 189      -4.185  -8.749  -6.180  1.00  0.00              
ATOM    256  OE2 GLU A 189      -4.705  -6.531  -6.140  1.00  0.00              
ATOM    257  C   GLU A 189      -0.240  -8.026  -2.824  1.00  0.00              
TER     258      GLU A 189 
END   
