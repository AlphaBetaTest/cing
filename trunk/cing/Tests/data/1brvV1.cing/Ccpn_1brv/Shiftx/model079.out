# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0591
  171   V  CA      60.4548
  171   V  CB      34.0853
  171   V  C      175.8977
  171   V  H        8.0549
  171   V  HA       4.0886
  171   V  HB       2.1059
  171   V  HG11     0.9421
  171   V  HG12     0.9143
  171   V  HG13     0.8472
  171   V  HG21     0.9956
  171   V  HG22     0.9620
  171   V  HG23     0.9831
  172   P  CA      62.8100
  172   P  CB      31.4642
  172   P  C      176.3421
  172   P  HA       4.1115
  172   P  HB2      2.2668
  172   P  HB3      2.0318
  172   P  HD2      3.6202
  172   P  HD3      3.8053
  172   P  HG2      2.1374
  172   P  HG3      2.2068
  173   C  N      120.7573
  173   C  CA      60.1054
  173   C  CB      40.8561
  173   C  C      176.1114
  173   C  H        8.8691
  173   C  HA       4.3177
  173   C  HB2      2.9662
  173   C  HB3      3.1386
  174   S  N      114.6661
  174   S  CA      60.8936
  174   S  CB      62.4644
  174   S  C      175.5166
  174   S  H        8.1966
  174   S  HA       4.0084
  174   S  HB2      4.0204
  174   S  HB3      4.0160
  175   T  N      111.5136
  175   T  CA      62.7857
  175   T  CB      69.5863
  175   T  C      175.8953
  175   T  H        7.6506
  175   T  HA       4.3881
  175   T  HB       4.2446
  175   T  HG21     1.2143
  175   T  HG22     1.2169
  175   T  HG23     1.2157
  176   C  N      119.5655
  176   C  CA      58.3128
  176   C  CB      40.8614
  176   C  C      173.6687
  176   C  H        8.1330
  176   C  HA       4.3484
  176   C  HB2      2.9890
  176   C  HB3      3.2133
  177   E  N      116.4448
  177   E  CA      57.5626
  177   E  CB      26.9023
  177   E  C      175.4200
  177   E  H        8.9481
  177   E  HA       3.5431
  177   E  HB2      2.1330
  177   E  HB3      1.9929
  177   E  HG2      2.2530
  177   E  HG3      2.3087
  178   G  N      104.5752
  178   G  CA      45.2154
  178   G  C      173.4214
  178   G  H        8.3978
  178   G  HA    -666.0000
  178   G  HA2      3.9597
  178   G  HA3      3.9594
  179   N  N      118.9912
  179   N  CA      52.3798
  179   N  CB      39.4736
  179   N  C      175.2888
  179   N  H        7.8598
  179   N  HA       4.6075
  179   N  HB2      2.8432
  179   N  HB3      2.7285
  179   N  HD21     7.1486
  179   N  HD22     6.4583
  180   L  N      129.1538
  180   L  CA      57.3264
  180   L  CB      41.6998
  180   L  C      179.4324
  180   L  H        8.4690
  180   L  HA       3.9644
  180   L  HB2      1.7239
  180   L  HB3      1.7013
  180   L  HD11     0.9185
  180   L  HD12     0.9181
  180   L  HD13     0.9185
  180   L  HD21     0.2298
  180   L  HD22     2.0689
  180   L  HD23     0.8671
  180   L  HG       1.7293
  181   A  N      120.6753
  181   A  CA      55.2025
  181   A  CB      18.6535
  181   A  C      179.7164
  181   A  H        7.9073
  181   A  HA       3.9233
  181   A  HB1      1.3872
  181   A  HB2      1.3721
  181   A  HB3      1.3339
  182   C  N      116.0631
  182   C  CA      60.1176
  182   C  CB      43.5002
  182   C  C      175.8308
  182   C  H        7.9472
  182   C  HA       4.1091
  182   C  HB2      3.0311
  182   C  HB3      3.2225
  183   L  N      119.0422
  183   L  CA      57.9998
  183   L  CB      41.1921
  183   L  C      179.5297
  183   L  H        7.9391
  183   L  HA       3.9314
  183   L  HB2      1.8622
  183   L  HB3      1.7218
  183   L  HD11     0.9177
  183   L  HD12     0.9896
  183   L  HD13     0.9020
  183   L  HD21     0.4426
  183   L  HD22     2.3116
  183   L  HD23     0.8681
  183   L  HG       1.7025
  184   S  N      114.2381
  184   S  CA      61.4470
  184   S  CB      62.8749
  184   S  C      176.3581
  184   S  H        7.8259
  184   S  HA       4.2739
  184   S  HB2      4.0234
  184   S  HB3      3.9108
  185   L  N      120.2638
  185   L  CA      56.1179
  185   L  CB      41.5355
  185   L  C      175.0474
  185   L  H        7.6277
  185   L  HA       4.1630
  185   L  HB2      1.5820
  185   L  HB3      1.7109
  185   L  HD11     0.9164
  185   L  HD12     0.9170
  185   L  HD13     0.9166
  185   L  HD21     0.3527
  185   L  HD22     2.7904
  185   L  HD23     0.8621
  185   L  HG       1.7781
  186   C  N      112.3961
  186   C  CA      55.1002
  186   C  CB      42.5861
  186   C  C      172.4345
  186   C  H        7.0969
  186   C  HA       4.5040
  186   C  HB2      3.2095
  186   C  HB3      3.3383
  187   H  N      117.2903
  187   H  CA      54.4563
  187   H  CB      28.9886
  187   H  C      175.5131
  187   H  H        8.3566
  187   H  HA       4.6039
  187   H  HB2      3.1412
  187   H  HB3      3.3012
  187   H  HD2      5.8804
  187   H  HE1      6.7867
  188   I  N      128.9933
  188   I  CA      60.4220
  188   I  CB      36.5016
  188   I  C      176.3578
  188   I  H        8.2675
  188   I  HA       4.0012
  188   I  HB       1.8712
  188   I  HD11     0.8499
  188   I  HD12     0.8444
  188   I  HD13     0.8396
  188   I  HG12     1.2692
  188   I  HG13     0.3941
  188   I  HG21     0.9136
  188   I  HG22     0.9123
  188   I  HG23     0.9107
 *189   E  N      120.4111
 *189   E  CA      55.9624
 *189   E  CB      30.2526
 *189   E  C      173.5672
 *189   E  H        8.5103
 *189   E  HA       4.1609
 *189   E  HB2      1.7375
 *189   E  HB3      1.7688
 *189   E  HG2      2.1331
 *189   E  HG3      2.3797
